Example/package.json

```diff
@@ -3,22 +3,17 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
     "react": "16.5.0",
     "react-native": "0.57.1",
-    "react-native-shimmer": "*"
+    "react-native-shimmer": "../"
```

Comment:
This is problematic as it is self referential and nowadays `npm`/`yarn` will not exclude ignored files from a local install.

Reply:
Ah, I see -- I was running into problems with the * with `yarn`. I'll revert this back and double check it works.

Reply:
* will just take whatever version it can find on npm so it's not very useful for local development no :-) This one is mostly for people trying out the project tho. Could write some script to update the dependency from local files for development purposes.

Reply:
Ah, that's what I thought. It seemed npm/yarn handled the one-directory-up file dependency for local development. I think I had to do a `yarn link` / `yarn link react-native-shimmer`, as well. I agree it's simpler this way for people trying out the sample app, though.